package.json

```diff
@@ -28,7 +28,6 @@
     "@grpc/grpc-js": "^1.8.22",
     "@grpc/proto-loader": "^0.7.2",
     "@hapi/boom": "^10.0.0",
-    "@puppeteer/browsers": "^1.6.0",
```

Comment:
I removed this to use the dependency included inside Puppeteer core. I couldn't find a reason why we were pinning this dependency here.